server/db/db.js

```diff
@@ -0,0 +1,7 @@
+const Sequelize = require('sequelize');
+const db = new Sequelize(
+  process.env.DATABASE_URL || 'postgres://localhost:5432/escapehatch', {
```

Comment:
PS Bones reads the title from package.json and uses that to construct a db… just sayin'